include/openthread/border_routing.h

```diff
@@ -498,6 +498,20 @@ void otBorderRoutingDhcp6PdSetEnabled(otInstance *aInstance, bool aEnabled);
  */
 otBorderRoutingDhcp6PdState otBorderRoutingDhcp6PdGetState(otInstance *aInstance);
 
+/**
+ * Process a prefix received from the DHCPv6 PD Server. The prefix is received on
+ * the DHCPv6 PD client callback and provided to the Routing Manager via this
+ * API.
+ *
+ * `OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE` must be enabled.
+ * The lifetime of the prefix can be updated by calling the function again with updated
+ * time values.
+ *
+ * @param[in] aInstance   A pointer to an OpenThread instance.
+ * @param[in] aPrefixInfo A pointer to the prefix information structure
+ */
+void otBorderRoutingDhcp6PdProcessPrefix(otInstance *aInstance, otBorderRoutingPrefixTableEntry *aPrefixInfo);
```

Comment:
Thanks @marius-preda.

I have not looked at the changes in this PR in detail.

- I know we have `otPlatBorderRoutingProcessIcmp6Ra()` mechasnim related to DHCP6 PD which calls to `RoutingManager::PdPrefixManager::ProcessPlatformGeneratedRa()` to determine the prefix.
- If I am not mistaken the new API adds another way of providing the prefix, correct? If so, do we need the new model, can we utilize existing approach?

cc @erjiaqing for his thoughts as well?

Reply:
Please ignore my question. I saw you mentioned this already in the PR description.